CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing guidelines
+
+As a contributor, here are the guidelines we would like you to follow:
+
+- Contributing guidelines
+  - [Code of conduct](#code-of-conduct)
+  - [Reporting Bugs](#reporting-bugs)
+  - [Feature Requests](#feature-requests)
+  - [PR Submission](#pr-submission)
+    - [Docs + Examples](#docs-and-examples)
```

Comment:
fix indentation 
```suggestion
  - [Docs + Examples](#docs-and-examples)
```

Reply:
actually it is right, it is a subsection of PR Submission